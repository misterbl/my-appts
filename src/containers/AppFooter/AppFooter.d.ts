import { FormikProps } from 'formik';
import { RouteComponentProps, StaticContext } from 'react-router';
import { InjectedIntlProps } from 'react-intl';

export interface IAppFooterState {
  clicked: string | null;
}
export type TAppFooterComponent = InjectedIntlProps &
  RouteComponentProps<any, StaticContext>;
