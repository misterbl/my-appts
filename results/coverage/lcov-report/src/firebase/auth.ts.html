<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/firebase/auth.ts</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">src/firebase</a> auth.ts
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/25</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/8</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/24</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js"><span class="cstat-no" title="statement not covered" >import { auth, provider } from './firebase';</span>
&nbsp;
// Sign Up
export const <span class="cstat-no" title="statement not covered" >doCreateUserWithEmailAndPassword = <span class="fstat-no" title="function not covered" >(</span>email: string, password: string) =&gt;</span>
<span class="cstat-no" title="statement not covered" >  auth.createUserWithEmailAndPassword(email, password);</span>
&nbsp;
  // Sign In
export const <span class="cstat-no" title="statement not covered" >doSignInWithEmailAndPassword = <span class="fstat-no" title="function not covered" >(</span>email: string, password: string) =&gt;</span>
<span class="cstat-no" title="statement not covered" >auth.signInWithEmailAndPassword(email, password);</span>
&nbsp;
// Sign out
export const <span class="cstat-no" title="statement not covered" >doSignOut = <span class="fstat-no" title="function not covered" >() =&gt;</span></span>
<span class="cstat-no" title="statement not covered" >  auth.signOut();</span>
&nbsp;
&nbsp;
  // Password Reset
export const <span class="cstat-no" title="statement not covered" >doPasswordReset = <span class="fstat-no" title="function not covered" >(</span>email: string) =&gt;</span>
<span class="cstat-no" title="statement not covered" >auth.sendPasswordResetEmail(email);</span>
&nbsp;
// Password Change
export const <span class="cstat-no" title="statement not covered" >doPasswordUpdate = <span class="fstat-no" title="function not covered" >(</span>password: string) =&gt;</span>
<span class="cstat-no" title="statement not covered" >auth &amp;&amp; auth.currentUser &amp;&amp; auth.currentUser.updatePassword(password);</span>
&nbsp;
// Facebook
export const <span class="cstat-no" title="statement not covered" >doFacebookSignIn = <span class="fstat-no" title="function not covered" >() =&gt; <span class="cstat-no" title="statement not covered" ></span>auth.signInWithPopup(provider).then(<span class="fstat-no" title="function not covered" >function(</span>result) {</span></span>
    // This gives you a Facebook Access Token. You can use it to access the Facebook API.
    const token = <span class="cstat-no" title="statement not covered" >result;</span>
    // The signed-in user info.
    const user = <span class="cstat-no" title="statement not covered" >result.user;</span>
<span class="cstat-no" title="statement not covered" >    console.log("user", user);</span>
<span class="cstat-no" title="statement not covered" >    console.log("token", token);</span>
    
    
    // ...
  }).catch(<span class="fstat-no" title="function not covered" >function(</span>error) {
    // Handle Errors here.
    const errorCode = <span class="cstat-no" title="statement not covered" >error.code;</span>
    const errorMessage = <span class="cstat-no" title="statement not covered" >error.message;</span>
<span class="cstat-no" title="statement not covered" >    console.log("errorCode", errorCode);</span>
<span class="cstat-no" title="statement not covered" >    console.log("errorMessage", errorMessage);</span>
    
    // The email of the user's profile used.
    const email = <span class="cstat-no" title="statement not covered" >error.email;</span>
<span class="cstat-no" title="statement not covered" >    console.log("email", email);</span>
    
    // The firebase.auth.AuthCredential type that was used.
    const credential = <span class="cstat-no" title="statement not covered" >error.credential;</span>
<span class="cstat-no" title="statement not covered" >    console.log("credential", credential);</span>
    
    // ...
  });</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Oct 23 2018 10:44:38 GMT+0100 (BST)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
