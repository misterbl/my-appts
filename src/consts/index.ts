import ROUTES from './routes';
import QUERIES from './queries';

export { ROUTES, QUERIES }