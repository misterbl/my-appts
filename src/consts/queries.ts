const queries = (args: any) => ({
  ADD_USER: `mutation {
    addUser (email: "${args.email}", avatar:"${args.avatar}" ) {
      _id
      firstName
      lastName
      avatar
      email
      address
      profileTitle
      profileDescription
     
    }
  }`,
  GET_USER: `mutation {
    getUser (email: "${args.email}") {
      _id
      firstName
      lastName
      avatar
      email
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  GET_USER_BY_ID: `mutation {
    getUserById (_id: "${args.id}") {
      _id
      firstName
      lastName
      avatar
      email
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  GET_USER_INFO: `mutation {
    getUser (email: "${args.email}") {
      _id
      firstName
      lastName
      avatar
      email
      address
    }
  }`,
  GET_AD_INFO: `mutation {
    getUser (email: "${args.email}") {
      _id
      profileTitle
      profileDescription
    }
  }`,
  UPDATE_AD_INFO: `mutation {
    updateUser (_id:"${args._id}" profileTitle: "${
    args.profileTitle
  }", profileDescription: "${args.profileDescription}") {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{name, dob, information, school, gender} 
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  UPDATE_PERSONAL_INFO: `mutation {
    updateUser (_id: "${args._id}", firstName: "${
    args.firstName
  }", lastName: "${args.lastName}", address: "${args.address}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,
  UPDATE_AVATAR: `mutation {
    updateUser (_id: "${args._id}", avatar:"${args.avatar}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,
  UPDATE_LOCATION: `mutation {
    updateUser (_id: "${args._id}", address: "${args.address}", lat: "${
    args.lat
  }", lng: "${args.lng}" ) {
      _id
      firstName
      lastName
      avatar
      address
      lat
      lng
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}

    }
  }`,

  UPDATE_ABOUT_YOU: `mutation {
    updateUser (_id: "${args._id}", drivingLicense: ${
    args.drivingLicense
  }, nonSmoker: ${args.nonSmoker}, car:${args.car}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      children{name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability

    }
  }`,
  UPDATE_AVAILABILITY: `mutation {
    updateUser (_id: "${args._id}", availability: ${JSON.stringify(
    JSON.stringify(args.availability),
  )}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      unavailability
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
    }
  }`,
  UPDATE_UNAVAILABILITY: `mutation {
    updateUser (_id: "${args._id}", unavailability: ${JSON.stringify(
    args.unavailability,
  )}) {
      _id
      firstName
      lastName
      avatar
      address
      drivingLicense
      nonSmoker
      car
      profileTitle
      profileDescription
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  ADD_CHILD: `mutation {
    updateUser (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
    }
  }`,
  UPDATE_CHILD: `mutation {
    updateChild (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
      availability{beforeSchool, morning, noon, afternoon, afterSchool, evening, night}
      unavailability
    }
  }`,
  REMOVE_CHILD: `mutation {
    removeChild (_id: "${args._id}", child: ${JSON.stringify(
    JSON.stringify(args.child),
  )}
) {
      _id
      firstName
      lastName
      avatar
      address
      profileTitle
      profileDescription
      children{id, name, dob, information, school, gender}
    }
  }`,
});

export default queries;
