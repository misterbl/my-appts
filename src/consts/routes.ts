const routes = {
  INDEX: '/',
  CARD: '/card',
  SIGN_IN: '/signin',
  REGISTER: '/register',
  DASHBOARD: './dashboard',
  INBOX: './inbox',
  FAVOURITE: './favourite',
  SEARCH: './search',
};

export default routes;
