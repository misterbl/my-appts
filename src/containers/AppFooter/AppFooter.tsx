import * as React from 'react';
// import { Formik, Form } from 'formik';
// import { auth } from '../../firebase';
import { injectIntl } from 'react-intl';
import { withRouter } from 'react-router';
import MessageIcon from '../../styles/assets/messageIcon'
import { IDashBoardComponent } from './AppFooter.d';
// import ROUTES from '../../consts/routes';

export class DashBoard extends React.Component<IDashBoardComponent> {
  
  render() {
    // const { formatMessage } = this.props.intl;

    return (
      <div className="flex absolute bottom-0 mb2">
       <span>APP FOOTER</span> 
      <MessageIcon/>


      </div>
    );
  }
}

const injectIntlDashBoard = injectIntl(DashBoard);
export default withRouter(injectIntlDashBoard);
