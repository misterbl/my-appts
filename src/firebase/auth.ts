import { auth, provider } from './firebase';

// Sign Up
export const doCreateUserWithEmailAndPassword = (email: string, password: string) =>
  auth.createUserWithEmailAndPassword(email, password);

  // Sign In
export const doSignInWithEmailAndPassword = (email: string, password: string) =>
auth.signInWithEmailAndPassword(email, password);

// Sign out
export const doSignOut = () =>
  auth.signOut();


  // Password Reset
export const doPasswordReset = (email: string) =>
auth.sendPasswordResetEmail(email);

// Password Change
export const doPasswordUpdate = (password: string) =>
auth && auth.currentUser && auth.currentUser.updatePassword(password);

// Facebook
export const doFacebookSignIn = () => auth.signInWithPopup(provider).then(function(result) {
    // This gives you a Facebook Access Token. You can use it to access the Facebook API.
    const token = result;
    // The signed-in user info.
    const user = result.user;
    console.log("user", user);
    console.log("token", token);
    
    
    // ...
  }).catch(function(error) {
    // Handle Errors here.
    const errorCode = error.code;
    const errorMessage = error.message;
    console.log("errorCode", errorCode);
    console.log("errorMessage", errorMessage);
    
    // The email of the user's account used.
    const email = error.email;
    console.log("email", email);
    
    // The firebase.auth.AuthCredential type that was used.
    const credential = error.credential;
    console.log("credential", credential);
    
    // ...
  });