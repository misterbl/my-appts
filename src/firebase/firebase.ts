import * as firebase from 'firebase/app';
import 'firebase/auth';


const config = {
    apiKey: "AIzaSyAUFtb8M2Pzfgpn48-F736NVgKlCcc2haQ",
    authDomain: "kidappi-app.firebaseapp.com",
    databaseURL: "https://kidappi-app.firebaseio.com",
    projectId: "kidappi-app",
    storageBucket: "kidappi-app.appspot.com",
    messagingSenderId: "600523020740"
  };

  if (!firebase.apps.length) {
    firebase.initializeApp(config);
  }

  const auth = firebase.auth();
  auth.useDeviceLanguage();
  const provider = new firebase.auth.FacebookAuthProvider()
export {
  auth,
  provider,
};