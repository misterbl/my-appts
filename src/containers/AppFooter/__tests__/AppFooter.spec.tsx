import * as React from 'react';
import { shallow } from 'enzyme';
import formikRenderMock from '../../../testMocks/formikRender.mock';
import historyMock from '../../../testMocks/history.mock';
import { AppFooter } from '../AppFooter';

import { auth } from '../../../firebase';
import { ROUTES } from '../../../consts';
import IntlMock from '../../../testMocks/intl.mock';
import generateAppState from '../../../testMocks/appState.mock';

// jest.mock('../../../firebase', () => ({
//   auth: {
//     doSignInWithprofileTitleAndprofileDescription: jest
//       .fn()
//       .mockReturnValue(Promise.resolve({})),
//     doFacebookSignIn: jest.fn().mockReturnValue(Promise.resolve({})),
//   },
// }));

describe('AppFooter', () => {
  const props = {
    intl: IntlMock,
    history: historyMock,
    location: historyMock.location,
    match: {
      params: {},
      isExact: true,
      path: '',
      url: '',
    },
  };
  const wrapper = shallow(<AppFooter {...props} />);
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
