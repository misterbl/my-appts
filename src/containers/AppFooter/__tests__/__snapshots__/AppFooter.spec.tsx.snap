// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppFooter
    history={
        Object {
            "action": "PUSH",
            "block": [Function],
            "createHref": [Function],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "length": 123,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "key": "",
              "pathname": "",
              "search": "",
              "state": "",
            },
            "push": [Function],
            "replace": [Function],
          }
    }
    intl={
        Object {
            "defaultFormats": "",
            "defaultLocale": "",
            "formatDate": [Function],
            "formatHTMLMessage": [Function],
            "formatMessage": [Function],
            "formatNumber": [Function],
            "formatPlural": [Function],
            "formatRelative": [Function],
            "formatTime": [Function],
            "formats": "",
            "locale": "",
            "messages": Object {},
            "now": [Function],
          }
    }
    location={
        Object {
            "hash": "",
            "key": "",
            "pathname": "",
            "search": "",
            "state": "",
          }
    }
    match={
        Object {
            "isExact": true,
            "params": Object {},
            "path": "",
            "url": "",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Svg
          Icon={[Function]}
          fill={undefined}
          handleClick={[Function]}
          name={
                    <FormattedMessage
                              id="content|appfooter|inbox"
                              values={Object {}}
                    />
          }
/>,
        <Svg
          Icon={[Function]}
          fill={undefined}
          handleClick={[Function]}
          name={
                    <FormattedMessage
                              id="content|appfooter|dashboard"
                              values={Object {}}
                    />
          }
/>,
        <Svg
          Icon={[Function]}
          fill={undefined}
          handleClick={[Function]}
          name={
                    <FormattedMessage
                              id="content|appfooter|favourite"
                              values={Object {}}
                    />
          }
/>,
        <Svg
          Icon={[Function]}
          fill={undefined}
          handleClick={[Function]}
          name={
                    <FormattedMessage
                              id="content|appfooter|search"
                              values={Object {}}
                    />
          }
/>,
        <Svg
          Icon={[Function]}
          fill={undefined}
          handleClick={[Function]}
          name={
                    <FormattedMessage
                              id="content|appfooter|booking"
                              values={Object {}}
                    />
          }
/>,
      ],
      "className": "flex bg-white justify-around fixed w-100 bottom-0 pv2 ph3 shadow-1",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "/inbox",
        "nodeType": "function",
        "props": Object {
          "Icon": [Function],
          "fill": undefined,
          "handleClick": [Function],
          "name": <FormattedMessage
            id="content|appfooter|inbox"
            values={Object {}}
/>,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "/dashboard",
        "nodeType": "function",
        "props": Object {
          "Icon": [Function],
          "fill": undefined,
          "handleClick": [Function],
          "name": <FormattedMessage
            id="content|appfooter|dashboard"
            values={Object {}}
/>,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "/favourite",
        "nodeType": "function",
        "props": Object {
          "Icon": [Function],
          "fill": undefined,
          "handleClick": [Function],
          "name": <FormattedMessage
            id="content|appfooter|favourite"
            values={Object {}}
/>,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "/search",
        "nodeType": "function",
        "props": Object {
          "Icon": [Function],
          "fill": undefined,
          "handleClick": [Function],
          "name": <FormattedMessage
            id="content|appfooter|search"
            values={Object {}}
/>,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "/booking",
        "nodeType": "function",
        "props": Object {
          "Icon": [Function],
          "fill": undefined,
          "handleClick": [Function],
          "name": <FormattedMessage
            id="content|appfooter|booking"
            values={Object {}}
/>,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Svg
            Icon={[Function]}
            fill={undefined}
            handleClick={[Function]}
            name={
                        <FormattedMessage
                                    id="content|appfooter|inbox"
                                    values={Object {}}
                        />
            }
/>,
          <Svg
            Icon={[Function]}
            fill={undefined}
            handleClick={[Function]}
            name={
                        <FormattedMessage
                                    id="content|appfooter|dashboard"
                                    values={Object {}}
                        />
            }
/>,
          <Svg
            Icon={[Function]}
            fill={undefined}
            handleClick={[Function]}
            name={
                        <FormattedMessage
                                    id="content|appfooter|favourite"
                                    values={Object {}}
                        />
            }
/>,
          <Svg
            Icon={[Function]}
            fill={undefined}
            handleClick={[Function]}
            name={
                        <FormattedMessage
                                    id="content|appfooter|search"
                                    values={Object {}}
                        />
            }
/>,
          <Svg
            Icon={[Function]}
            fill={undefined}
            handleClick={[Function]}
            name={
                        <FormattedMessage
                                    id="content|appfooter|booking"
                                    values={Object {}}
                        />
            }
/>,
        ],
        "className": "flex bg-white justify-around fixed w-100 bottom-0 pv2 ph3 shadow-1",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "/inbox",
          "nodeType": "function",
          "props": Object {
            "Icon": [Function],
            "fill": undefined,
            "handleClick": [Function],
            "name": <FormattedMessage
              id="content|appfooter|inbox"
              values={Object {}}
/>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "/dashboard",
          "nodeType": "function",
          "props": Object {
            "Icon": [Function],
            "fill": undefined,
            "handleClick": [Function],
            "name": <FormattedMessage
              id="content|appfooter|dashboard"
              values={Object {}}
/>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "/favourite",
          "nodeType": "function",
          "props": Object {
            "Icon": [Function],
            "fill": undefined,
            "handleClick": [Function],
            "name": <FormattedMessage
              id="content|appfooter|favourite"
              values={Object {}}
/>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "/search",
          "nodeType": "function",
          "props": Object {
            "Icon": [Function],
            "fill": undefined,
            "handleClick": [Function],
            "name": <FormattedMessage
              id="content|appfooter|search"
              values={Object {}}
/>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "/booking",
          "nodeType": "function",
          "props": Object {
            "Icon": [Function],
            "fill": undefined,
            "handleClick": [Function],
            "name": <FormattedMessage
              id="content|appfooter|booking"
              values={Object {}}
/>,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
